src/env.js

```diff
@@ -203,13 +202,18 @@ config.monitors = config.monitors
 
 config.reports = (config.reports || [])
     .map(item => {
-
+        if (item.file === "reportEmail") {
+            if (typeof item.params.smtp.auth === 'undefined' || item.params.smtp.auth === null) {
+                item.params.smtp.auth = {};
+            }
+            item.params.smtp.auth.user = process.env.SMTPU;
+            item.params.smtp.auth.pass = process.env.SMTPP;
```

Comment:
Looking at this again it looks like we're just overriding `item.params.smtp.auth` with whatever is in `env`. If that's the case, is there any reason we couldn't just do:
```
item.params.smtp.auth = {
  user: process.env.SMTPU,
  pass: process.env.SMTPP
};
```
rather than all the null/undef checks?

Reply:
ok sure